clients/fides-js/src/components/tcf/TcfTabs.tsx

```diff
@@ -98,25 +98,27 @@ const TcfTabs = ({
             exercise you consent for each vendor based on the legal basis they
             assert.
           </InfoBox>
-          <AllOnOffButtons
-            enabledIds={enabledIds}
-            onChange={onChange}
-            modelTypeMappings={{
-              vendorsConsent: [
-                ...(experience.tcf_vendor_consents || []),
-                ...(experience.tcf_system_consents || []),
-              ],
-              vendorsLegint: [
-                ...(experience.tcf_vendor_legitimate_interests || []),
-                ...(experience.tcf_system_legitimate_interests || []),
-              ],
-            }}
-          />
           <TcfVendors
             experience={experience}
             enabledVendorConsentIds={enabledIds.vendorsConsent}
             enabledVendorLegintIds={enabledIds.vendorsLegint}
             onChange={handleUpdateDraftState}
+            allOnOffButtons={
```

Comment:
I needed the buttons to be after the vendor filter so I defined it here so I could pull the `onChange` from this context. Not sure if this is the best approach.

Reply:
hmm this is reasonable but yeah doesn't feel great. this is where things like redux are very helpful, alas! I haven't been able to think of a better solution yet, so let's go with this one 👍